Source/ARTWebSocketTransport.m

```diff
@@ -57,7 +57,9 @@ + (void)setWebSocketClass:(const Class)webSocketClass {
     configuredWebsocketClass = webSocketClass;
 }
 
-- (instancetype)initWithRest:(ARTRestInternal *)rest options:(ARTClientOptions *)options resumeKey:(NSString *)resumeKey connectionSerial:(NSNumber *)connectionSerial {
+- (instancetype)initWithRest:(ARTRestInternal *)rest
+                     options:(ARTClientOptions *)options
+                   resumeKey:(NSString *)resumeKey recoveryKey:(nullable NSString *)recoveryKey {
```

Comment:
For consistency and readability.
Also because `nullable` means nothing to the compiler when used on implementation side (it means something only in the interface declaration -- hopefully my memory is not hazy on this point as it's been a while since I puzzled on this one 😅).
And, finally, because these are new args there is no reason not to make them `const`.

```suggestion
                   resumeKey:(NSString *const)resumeKey
                 recoveryKey:(NSString *const)recoveryKey {
```